BuildToolsVersion.txt

```diff
@@ -1 +1 @@
-1.0.25-prerelease-00420-01
\ No newline at end of file
+1.0.25-prerelease-00424-03
```

Comment:
CoreFx is now up to 00425-03.  We should probably update to match.  But that might be handled by #1213 if it gets merged first